code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -158,3 +158,93 @@ var/last_chew = 0
 
 /obj/item/weapon/handcuffs/cyborg
 	dispenser = 1
+
+
+/obj/item/weapon/handcuffs/ziptie
+	name = "zipties"
+	desc = "Sturdy, and reliable plastic zipties for binding the wrists."
+	gender = PLURAL
+	icon = 'icons/obj/items.dmi'
+	breakouttime = 600 //One minute, because they're not made of steel, therefore easier to slip out of.
+	icon_state = "ziptie"
+	cuff_sound = 'sound/weapons/cablecuff.ogg'
+	var/singular_name = "ziptie"
+	var/amount = 6
+	var/max_amount = 6
+
+/obj/item/weapon/handcuffs/ziptie/New(var/loc, var/amount=null)
+	..()
+	if (amount)
+		src.amount=amount
+	return
+
+/obj/item/weapon/handcuffs/ziptie/Del()

```

Comment:
Actually. Scratch that. I'm not seeing anything that would OPEN "window=stack" here. So you can probably quite safely remove this proc (lines 181 - 184 inclusive).